en/docs/learn/integration-tutorials/transforming-message-content.md

```diff
@@ -195,39 +126,121 @@ Let's create an Endpoint to represent the Hospital Service back-end service.
                 Endpoint HTTP REST Method.
             </td>
         </tr>
+      </tbody>
+    </table>
+
+    <a href="{{base_path}}/assets/img/learn/tutorials/transforming-message-content/endpoint-artifact.png"><img src="{{base_path}}/assets/img/learn/tutorials/transforming-message-content/endpoint-artifact.png" alt="endpoint artifact" width="80%"></a>
+
+5.  Click **Create**.
+
+#### Create a REST API
+
+1. Go to **MI Project Explorer** > **APIs**.
+
+    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/create-rest-api.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/create-rest-api.png" alt="create new api" width="30%"></a>
+
+2. Hover over **APIs** and click the **+** icon that appears to open the **API Form**.
+
+    <a href="{{base_path}}/assets/img/learn/tutorials/add-api.png"><img src="{{base_path}}/assets/img/learn/tutorials/add-api.png" alt="add API" width="30%"></a>
+
+3. Enter the details given below to create a new REST API.
+
+    <table>
+      <tr>
+        <th>Property</th>
+        <th>Value</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>Name</td>
+        <td><code>HealthcareAPI</code></td>
+        <td>
+          The name of the REST API.
+        </td>
+      </tr>
+      <tr>
+        <td>Context</td>
+        <td><code>/healthcare </code></td>
+        <td>
+          Here you are anchoring the API in the <code>/healthcare </code> context. This will become part of the name of the generated URL used by the client when sending requests to the Healthcare service. For example, setting the context to /healthcare means that the API will only handle HTTP requests where the URL path starts with <code>http://host:port/healthcare<code>.
```

Comment:
```suggestion
          Here you are anchoring the API in the <code>/healthcare </code> context. This will be included in the name of the generated URL that the client uses to send requests to the Healthcare service. For example, setting the context to /healthcare means that the API will only handle HTTP requests where the URL path starts with <code>http://host:port/healthcare<code>.
```